### DISCUSSION TITLE
What are the 'internal' files (paths, names) of a tex projevt to be deleted?

### DISCUSSION THREAD

**Discussion by gooly** [texstudio-org/texstudio#3461] [Q&A]
I (Win. 10, 64, TexStudio TeXstudio 4.3.1 (git 4.3.1) Qt-Version 6.3.1, kompiliert mit Qt 6.3.1 R)
would like to delete all traces of the tex-file (internal, hidden, logs) that I have written with TexStudio (MikTex). The files in the folder where I wrote it is already deleted. What else can be left?
I don't want to touch (delete, change,..) neither the templates nor any config. files.
Thanks and a happy New Year.

**Comment by sunderme** [comment]
texstudio saves session and caches some file content in %appdata%\texstudio
